variants/backend-base/config/environments/production.rb

```diff
@@ -46,10 +36,7 @@
            'ENV.fetch("RAILS_LOG_LEVEL", "info")',
            'ENV.fetch("RAILS_LOG_LEVEL", ENV.fetch("LOG_LEVEL", "info"))'
 
-gsub_file! "config/environments/production.rb",
-           "ActiveSupport::Logger.new(STDOUT)",
-           "ActiveSupport::Logger.new($stdout)"
-
+# TODO: https://github.com/ackama/rails-template/issues/569
 insert_into_file "config/environments/production.rb",
                  after: /.*config\.public_file_server\.enabled.*\n/ do
```

Comment:
issue: I don't see how this can match, because the Rails 8 config does not include this anymore by default

https://github.com/ackama/rails-template-variants-comparison/blob/main/config/environments/production.rb